dotnet/src/webdriver/SeleniumManager.cs

```diff
@@ -36,27 +36,25 @@ public static class SeleniumManager
     {
         private static readonly ILogger _logger = Log.GetLogger(typeof(SeleniumManager));
 
-        private static readonly string BinaryFullPath = Environment.GetEnvironmentVariable("SE_MANAGER_PATH");
-
         private static readonly JsonSerializerOptions _serializerOptions = new() { PropertyNameCaseInsensitive = true, TypeInfoResolver = SeleniumManagerSerializerContext.Default };
 
-        static SeleniumManager()
+        private static readonly Lazy<string> _lazyBinaryFullPath = new(() =>
         {
-
-            if (BinaryFullPath == null)
+            string? binaryFullPath = Environment.GetEnvironmentVariable("SE_MANAGER_PATH");
```

Comment:
Nullable reference types are not yet enabled in this project, sorry :( I understand you, but let's not produce new warnings.

Reply:
Nullable reference types were invented with this scenario in mind, and can be enabled file-by-file (or even more granular) with `#nullable enable`

Would it be fine to make those kinds of changes when we touch adjacent code, as long as it is not too invasive?

Reply:
Let's go with `#nullable enable` for this file, it is fine. This is how I am going to introduce this feature in BiDi namespace.

Reply:
I enabled NRT for this file, and fixed two hypothetical null reference exceptions with webdriver exceptions

> what nobody wants to do :)

:)

Reply:
If you don't mind I pushed changes regarding optional properties in json model. Are you OK with these changes?

Reply:
Unfortuanately the change to `ResultReponse` will not work. From the docs:
> The parameter names of a parameterized constructor must match the property names and types. Matching is case-insensitive, and the constructor parameter must match the actual property name even if you use [[JsonPropertyName]](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonpropertynameattribute) to rename a property.

Reply:
Additionally, this does not remove any nullability issues - this goes back to the null reference exception style. Any constructor parameter that is not found is populated with `default` ie. `null` for strings. This is changing in v9 actually, behind a feature flag: https://devblogs.microsoft.com/dotnet/system-text-json-in-dotnet-9/#respecting-non-optional-constructor-parameters

Reply:
I made one final push - to make the properties required and to suppress the null warning - with the knowledge that this only gets instantiated by JSON serialization

Reply:
Thank you, good collaboration.

> The parameter names of a parameterized constructor must match the property names and types. Matching is case-insensitive, and the constructor parameter must match the actual property name even if you use [[JsonPropertyName]](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonpropertynameattribute) to rename a property.

It worked though.